### PR TITLE
MM-44637_Site description removes "Log in" text from login screen

### PR DESCRIPTION
**Pull request by julmondragon**
#### Summary
Some updates to the Customization options for Login/Signup pages.

- Custom Brand Text:
  - Shows centered

- Site Description:
  - Help text updated to: "Displays as a title above the login form. When not specified, the phrase "Log in" is displayed."
  - Placeholder removed.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-44637

##### No Custom Brand Text
Screen Shot 2022-07-14 at 1 55 06 PM

##### Short Custom Brand Text
Screen Shot 2022-07-13 at 10 23 54 AM

##### Long Custom Brand Text
Screen Shot 2022-07-13 at 9 18 49 AM

#### Release Note
```release-note
- Custom Brand Text now shows centered.
- Site Description configuration does not show placeholder and help text now reads "Displays as a title above the login form. When not specified, the phrase "Log in" is displayed."
```

### CODE REVIEW

**Comment by julmondragon** [comment]
@cwarnermm Please be aware that documentation updates will be required due to the changes in this PR regarding Custom Brand Text and Site Description configs.
cc @marianunez

**Comment by cwarnermm** [comment]
Really appreciate the heads-up, @julmondragon! Thank you!

File: components/login/login.tsx [hunk 1]
```diff
@@ -659,8 +659,8 @@ const Login = ({onCustomizeHeader}: LoginProps) => {
     };
 
     const getMessageSubtitle = () => {
-        if (enableCustomBrand && CustomBrandText) {
-            return (
+        if (enableCustomBrand) {
+            return CustomBrandText && (
```
Comment (marianunez) [inline components/login/login.tsx:663]: Curious why this change? Seems to be have the same result as before?

**Review by marianunez — APPROVED** [review]
LGTM, thanks @julmondragon!

File: components/login/login.tsx (hunk 1 above)
Comment (julmondragon) [reply components/login/login.tsx:663]: Before it only entered here if both Custom Brand was enabled and Text was provided. Else the default text was always returned.
Now only having Custom Brand enabled it enters here, and no default text is returned.

**Review by pvev — APPROVED** [review]
LGTM! thanks @julmondragon :shipit:

**Review by matthewbirtch — CHANGES_REQUESTED** [review]
Thanks @julmondragon, this is looking good. A few things I discovered while playing with this. They may be beyond the original scope of this PR, but should be fixed:

1. As soon as I enabled custom branding, the text "Collaborate with your team in real-time" was removed on the login screen. I think this should only be removed if I have actually filled in the custom brand text.
image
2. I think we should only center the custom brand text if an image is added. Otherwise we end up with this
image
3. I just quickly tested a markdown headline in the custom brand text space and it appears that the alignment isn't getting applied properly. Headings are left-aligned while paragraph text is centered.
image

**Comment by julmondragon** [comment]
> Thanks @julmondragon, this is looking good. A few things I discovered while playing with this. They may be beyond the original scope of this PR, but should be fixed:
> ...

Updated

**Review by matthewbirtch — CHANGES_REQUESTED** [review]
Thanks @julmondragon. Those issues all look resolved now. However, I found a new one. If the custom brand text spans multiple lines it causes the "Don't have an account" line to move further away from the form container and it feels like it's no longer anchored to the form area. Is is possible to fix this so that it's always going to be the same margin from the top of the form container?

image

**Review by stevemudie — APPROVED** [review]
This looks good @julmondragon!
